test.js

```diff
@@ -20,4 +20,14 @@ it('should get article title from a HTML document', function () {
 	assertHTML('tylerhenkel', 'Angular Fullstack 1.2.0 available now');
 	assertHTML('numediaweb', 'Web App with dream team; AngularJS, Cordova, Yeoman & Topcoat');
 	assertHTML('github-repo', 'urls-md');
+	assertHTML('youtube-wiki', 'Wikipedia: #Edit2014');
+});
+
+it('should get title containing colon from Youtube', function(){
```

Comment:
I don't see the point in this. It's just duplicating the above one. Also not interested in requiring network to run the tests.